e2etests/web/regular_integration_tests/test_driver/platform_messages_integration.dart

```diff
@@ -46,14 +46,11 @@ void main() async {
     app.main();
     await tester.pumpAndSettle();
     final Map<String, dynamic> createArgs = <String, dynamic>{
-      'id': '567',
+      'id': 567,
       'viewType': 'MyView',
     };
     await SystemChannels.platform_views.invokeMethod<void>('create', createArgs);
-    final Map<String, dynamic> disposeArgs = <String, dynamic>{
-      'id': '567',
-    };
-    await SystemChannels.platform_views.invokeMethod<void>('dispose', disposeArgs);
```

Comment:
@ferhatb @yjbanov @hterkelsen 

I'm trying to run these tests on different modes. This error really confused me. Unless I change the arguments for `dispose` only with an integer, I'm getting the following stack trace:
```
Failure in method: Should create and dispose view embedder
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞═════════════════
The following TypeErrorImpl was thrown running a test:
Expected a value of type 'int', but got one of type
'LinkedMap<dynamic, dynamic>'

When the exception was thrown, this was the stack:
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 216:49      throw_
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/errors.dart 64:3        castError
dart-sdk/lib/_internal/js_dev_runtime/private/ddc_runtime/operations.dart 442:10  cast
dart-sdk/lib/_internal/js_dev_runtime/patch/core_patch.dart 204:12                as
lib/ui/ui.dart 139:13                                                             _disposePlatformView
lib/ui/ui.dart 108:7                                                              handlePlatformViewCall
lib/_engine/engine/platform_dispatcher.dart 380:11                                [_sendPlatformMessage]
lib/_engine/engine/platform_dispatcher.dart 216:5                                 sendPlatformMessage
lib/ui/src/ui/window.dart 116:24                                                  sendPlatformMessage
packages/flutter/src/services/system_channels.dart.js 2184:17                     [_sendPlatformMessage]
packages/flutter/src/services/system_channels.dart.js 2228:40                     send
packages/flutter_test/src/_matchers_web.dart.js 3264:40                           send
packages/flutter/src/services/system_channels.dart.js 938:50                      _invokeMethod
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 84:54                runBody
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 123:12               _async
packages/flutter/src/services/system_channels.dart.js 936:20                      [_invokeMethod]
packages/flutter/src/services/system_channels.dart.js 950:33                      invokeMethod
platform_messages_integration.dart.js 79:63                                       <fn>
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 45:50                <fn>
packages/stack_trace/src/stack_zone_specification.dart.js 163:98                  <fn>
packages/stack_trace/src/stack_zone_specification.dart.js 231:16                  [_run]
packages/stack_trace/src/stack_zone_specification.dart.js 163:80                  <fn>
dart-sdk/lib/async/zone.dart 1198:47                                              _rootRunUnary
dart-sdk/lib/async/zone.dart 1100:19                                              runUnary
dart-sdk/lib/async/future_impl.dart 143:18                                        handleValue
dart-sdk/lib/async/future_impl.dart 696:44                                        handleValueCallback
dart-sdk/lib/async/future_impl.dart 725:32                                        _propagateToListeners
dart-sdk/lib/async/future_impl.dart 529:5                                         [_completeWithValue]
dart-sdk/lib/async/future_impl.dart 567:7                                         <fn>
packages/stack_trace/src/stack_zone_specification.dart.js 231:16                  [_run]
packages/stack_trace/src/stack_zone_specification.dart.js 154:71                  <fn>
dart-sdk/lib/async/zone.dart 1190:13                                              _rootRun
dart-sdk/lib/async/zone.dart 1093:19                                              run
dart-sdk/lib/async/zone.dart 997:7                                                runGuarded
dart-sdk/lib/async/zone.dart 1037:23                                              callback
dart-sdk/lib/async/schedule_microtask.dart 41:11                                  _microtaskLoop
dart-sdk/lib/async/schedule_microtask.dart 50:5                                   _startMicrotaskLoop
dart-sdk/lib/_internal/js_dev_runtime/patch/async_patch.dart 166:15               <fn>

The test description was:
  Should create and dispose view embedder

```

However when I read the framework code the arguments are also send as a `<String, dynamic>` map: https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/platform_views.dart#L945  

Do you have any opinion on this type error? (I think there can be some other breakages in the code, if the ` <String, dynamic>` is the correct argument set but  started causing an error due to a recent change`)